engine/physics/src/physics/test/test_physics.cpp

```diff
@@ -117,6 +117,7 @@ Test3D::Test3D()
 , m_IsEnabledFunc(dmPhysics::IsEnabled3D)
 , m_SetEnabledFunc(dmPhysics::SetEnabled3D)
 , m_IsSleepingFunc(dmPhysics::IsSleeping3D)
+, m_WakeupFunc(dmPhysics::Wakeup3D)
```

Comment:
I forgot to double check this when the feature was reviewed.
If it's not implemented, we cannot use the null implementations as intended.

So, we add a unit test in order to make sure it's implemented in all variants.

Reply:
ping @otsakir: For future reference ☝️